### ISSUE TITLE
[Suggestion] Alternate coloring for integers and floats

### ISSUE THREAD

**Issue by kdschlosser** [SeeSharpSoft/intellij-csv-validator#211]
Provide alternate coloring when viewing integers and floats in table view. The colors to be used can be gotten from the IDE's editor settings. It would be something to beautify the table view.

**Comment by SeeSharpSoft** [comment]
@kdschlosser Thanks for the suggestion.

Yes, this would be quite neat - not only in the table editor.

**Comment by kdschlosser** [comment]
That is true. table view or raw view it would definitely reduce the single color boredom that a users eyes can have.
